containers.sh

```diff
@@ -64,7 +64,7 @@ todo_db(){
 log "Checking for existing networks..."
 for network in "${networks[@]}"
 do
-  if [ "$(docker network ls | grep "${network}")" ]; then
+  if [ "$(docker network ls | grep -w "${network}")" ]; then
```

Comment:
I see what you're trying buuuut

```shell
$ echo "appnetwork" | grep app -q -w; echo $?
1

$ echo "app-network" | grep app -q -w; echo $?
0
```
(exitcode 1 == error == no match, 0 == match)
Aka, while `-w` does indeed switch grep over to looking for whole words and not just substring matches - it defines a word that consists of "word characters" (locale dependent). So when grepping for "app", it will not match "appnetwork" (as it can not match the entire word) but it would match "app-network" as `-` is not a word-character, and thus grep will see "app-network" as two separate words.

While it is not perfect, the snippet I posted earlier should be more robust:

```shell
if ! docker network inspect "${network}" 1> /dev/null 2> /dev/null; then
    echo "* Could not detect a '${network}' network"
   # Create network here
fi
```

(that being said - don't worry about it for now :D)